src/BlockchainReader/DAO.ts

```diff
@@ -52,4 +51,12 @@ export class DAO {
       .toArray()
     return blockHeight
   }
+
+  readonly findHashByHeight: findHashByHeight = async (blockHeight: number) => {
+    const [{ blockHash = null } = {}] = await this.collection
+      .find({ blockHeight }, { projection: { blockHash: true, _id: 0 } })
+      .limit(1)
```

Comment:
We assume the chain is linear for now. This is not the case, but that will be addressed in the future.